src/particle/particle/utilities.py

```diff
@@ -35,8 +34,9 @@ def programmatic_name(name):
     return name.lstrip("_")
 
 
-def str_with_unc(value, upper, lower=None):
-    # type: (float, Optional[float], Optional[float]) -> str
+def str_with_unc(
+    value: float, upper: Optional[float], lower: Optional[float] = None
```

Comment:
Given the code should we not have ` upper: Optional[float] = None` instead?

Reply:
We force users to be explicit, I think. `upper=None` means no uncertainty, while `lower=None` means upper and lower are the same. But maybe it's better with the default. Feel free to change if you want, it's just our utility, but not related to the PR. :)

Reply:
Indeed not. I will try and remember to look in the future. Depends on how we use the function. Fine as is, it seems.